src/main/java/de/muenchen/isi/domain/service/email/SendWorkAssignmentInformationService.java

```diff
@@ -52,6 +58,46 @@ public void sendWorkAssignmentInformationAsync(
         this.sendWorkAssignmentInformation(abfrage, stateMachineEvent);
     }
 
+    /**
+     * Erstellt den Stadtbezirktext.
+     * @param verortung
+     * @return Stadtbezirktext zusammengesetzt aus Nummer und Name
+     */
+    protected String getTextStadtbezirke(VerortungModel verortung) {
+        if (verortung == null) {
+            return StringUtils.EMPTY;
+        }
+        return verortung
```

Comment:
Um den Code robuster zu gestlaten, kann der Aufruf in eine Utilklasse gepacklt werden. Somit wird eine potentielle Nullpointerexception verhindert.
```java
return CollectionUtils.emptyIfNull(verortung.getStadtbezirke())
    .stream().
   ...
```